src/FlightMap/Widgets/CameraPageWidget.qml

```diff
@@ -202,6 +203,23 @@ Column {
                     anchors.left:   parent.left
                     anchors.right:  parent.right
                     spacing:        _margins
+                    Row {
+                        visible:    _isCamera
+                        spacing:    ScreenTools.defaultFontPixelWidth
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        QGCLabel {
+                            text:           qsTr("Camera Selector:")
+                            width:          _labelFieldWidth
+                            anchors.verticalCenter: parent.verticalCenter
+                        }
+                        QGCComboBox {
+                            id:             cameraSelector
+                            model:          _isCamera ? _dynamicCameras.cameraLabels : []
+                            width:          _editFieldWidth
+                            onActivated:    _dynamicCameras.cameras = index
+                            currentIndex:   0
```

Comment:
The selector always shows the first item in the list of the camera
Changing it to `currentIndex:   _dynamicCameras.currentCamera` fixes the issue

Reply:
Same idea.

Thanks!